x/reward/module.go

```diff
@@ -24,6 +23,7 @@ import (
 var (
 	_ module.AppModule      = AppModule{}
 	_ module.AppModuleBasic = AppModuleBasic{}
+	_ module.HasGenesis     = AppModule{}
```

Comment:
By adding the two interfaces below, we can detect which interfaces will be changed in the future at compile time.

```
_ module.HasServices         = AppModule{}
_ appmodule.HasBeginBlocker = AppModule{}
```

Reply:
It also same to x/volunteer module
```
_ module.HasGenesis         = AppModule{}
_ module.HasServices        = AppModule{}
_ appmodule.HasBeginBlocker = AppModule{}
```
+ It need to change InitGenesis's return type

Reply:
+ _ module.HasConsensusVersion = AppModuleBasic{}

Reply:
https://github.com/xpladev/xpla/pull/154/commits/76a6661211f895f81c2c7a9dd4ff25a874613080